app/app.rb

```diff
@@ -0,0 +1,101 @@
+#ENV['RACK_ENV'] ||= 'development'
+
+require 'sinatra/base'
+require './app/data_mapper_setup'
+require 'bcrypt'
+require 'sinatra/flash'
+
+class Chitter < Sinatra::Base
+  register Sinatra::Flash
+  enable :sessions
+  set :session_secret, 'super secret'
+  use Rack::MethodOverride
+
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'peeps/index'
+  end
+
+  get '/peeps/new' do
+    erb :'peeps/new'
+  end
+
+  post '/peeps' do
+    Peep.create(message: params[:message], time_stamp: Time.now)
+    redirect to('/peeps')
+  end
+
+=begin
```

Comment:
Style/BlockComments: Do not use block comments. (https://github.com/bbatsov/ruby-style-guide#no-block-comments)